### ISSUE TITLE
[BUG] can't move cursor inside indented math block in Vim NORMAL mode (Latex Suite v1.8.1)

### ISSUE THREAD

**Issue by Deeperand** [artisticat1/obsidian-latex-suite#192]
# Description

If turn on `Setting` -> `Math popup preview`, can't move cursor inside (either by `j`, `k`, `h`, `l` or by  `<up>`, `<down>`, `<left>`, `<right>` key) in indented math block in Vim NORMAL mode if there are some Markdown list before it (but in INSERT mode I can move cursor). e.g
```markdown
<!-- can't move cursor inside the following block -->
- Markdown list item
    $$
    some math
    $$
```
Note here the math block must be indented.

# To Reproduce

- Turn on Vim mode in `Setting` -> `Editor` -> `Advanced` -> `Vim key bindings`;
- Turn on `Setting` -> `Latex Suite` -> `Math popup preview`;
- create a new note, copy the codes in above Markdown block into it;
- press `<esc>` to make sure you are in Vim NORMAL mode;
- put cursor in the first `$$`,  and press `j` or `<down>` try to move cursor down, and one will found cursor didn't move;
- use mouse click `some math` part, and one will found cursor can't be set into that line;

However, if turn off `Math popup preview`, the cursor can't move problem disappear.

# Expected Behavior

The cursor should freely move.

# Additional Info

I suspect this bug is raised from "Popup preview window for block math in Source mode" #138 in update v1.7.3, since when I use v1.7.2, I didn't meet this bug.

Besides, it looks like it will also affect snippet expanding behavior. I defined the following snippet for indented math block
```ts
{trigger: "mm", replacement: "    $$\n    $0\n    $$", options: "tA", description: "block-math"}
```
and if below Markdown list item, it will expand into
```markdown
- Markdown item 1
    $$
    
    $$m
```
However, in v1.7.2 or turn off `Math popup preview` in v1.8.1, it can expand correctly
```markdown
- Markdown item 1
    $$
    
    $$
```

In addition, I also found the condition to trigger this bug is a bit subtle. I found if append some math (either inline or block) before the Markdown list like (here I assume the following blocks is ALL CONTENT of the testing Markdown file)
```markdown
$math 1$
- Markdown item
    $$
    math 2
    $$
```
or
```markdown
$$
math 1
$$
- Markdown item
    $$
    math 2
    $$
```
then one can freely move cursor inside the both `math 2` block.

Platform Info
- Obsidian version: v.1.4.13
- `Latex Suite` version: v.1.8.1

**Comment by mzietlow** [comment]
I am experiencing something similar, where I can not insert newlines into math blocks. This is easily reproducable:

- Create a new vault.
- Install and enable Latex Suite.
- Insert `- $$$$`.
- Try creating a new line `- $$\n$$`. (Doesn't work).
- Disable Latex Suite.
- Try creating a new line `- $$\n$$`. (Does work).

**Comment by artisticat1** [comment]
Fixed in 1.8.2.
